package.json

```diff
@@ -50,7 +50,7 @@
   },
   "peerDependencies": {
     "backbone": "~1.3.3",
-    "underscore": "~1.8.3"
+    "underscore": ">= 1.8.3 < 2"
```

Comment:
I think this should be `< 1.9.x` underscore does not follow semver so `1.10` can be breaking with no warning.

Reply:
Ok, I will make that change.

Reply:
Maybe `<= 1.9.x` will have clearer intentions yet?

Reply:
Oh yep